orcid-core/src/main/java/org/orcid/core/manager/impl/RegistrationManagerImpl.java

```diff
@@ -290,7 +297,12 @@ private String createMinimalProfile(Registration registration, boolean usedCaptc
             name.setGivenNames(new GivenNames(registration.getGivenNames().getValue()));
         }
         recordNameManager.createRecordName(orcid, name);
-                
+
+        if (Features.EVENTS.isActive() && registration.getAffiliationForm() != null) {
```

Comment:
@DanielPalafox is this the right feature for this `Features.EVENTS`? I think this one is to track user events right?

Reply:
Yes, you are right I will put the correct togglz. Thanks